sgrep/lib/check_sgrep.ml

```diff
@@ -0,0 +1,26 @@
+
+(* for these languages, we are sure that $x is an error *)
+let lang_has_no_dollar_ids = Lang.(function
+  | Python
+  | Java
+  | Go
+  | C
+  | ML
+  -> true
+  | __else__
```

Comment:
I try to avoid the _ pattern, especially when there are a few cases, so in case we add stuff in lang.ml the compiler warns us of places to update.

Reply:
Good point, I'm of the same opinion but I overlooked it here. It actually detected that I was not using the same snapshot of Pfff as develop (for some reason to be investigated).